recipes/scipy/build.sh

```diff
@@ -1,9 +1,34 @@
 #!/bin/bash
 
-if [ `uname` == Darwin ]
-then
-    FCFLAGS=$CFLAGS
-    LDFLAGS="-undefined dynamic_lookup -bundle -Wl,-search_paths_first -L$(pwd) $LDFLAGS"
+
+# Letting SciPy set these for us.
+# This may not be the best long term strategy,
+# but it works fine to get our first build.
+unset LDFLAGS
+
+# Depending on our platform, shared libraries end with either .so or .dylib
+if [[ `uname` == 'Darwin' ]]; then
+    DYLIB_EXT=dylib
+else
+    DYLIB_EXT=so
 fi
 
+# If OpenBLAS is being used, we should be able to find the libraries.
+# As OpenBLAS now will have all symbols that BLAS or LAPACK have,
+# create libraries with the standard names that are linked back to
+# OpenBLAS. This will make it easy for NumPy to find it.
+test -f "${PREFIX}/lib/libopenblas.a" && ln -fs "${PREFIX}/lib/libopenblas.a" "${PREFIX}/lib/libblas.a"
+test -f "${PREFIX}/lib/libopenblas.a" && ln -fs "${PREFIX}/lib/libopenblas.a" "${PREFIX}/lib/liblapack.a"
+test -f "${PREFIX}/lib/libopenblas.${DYLIB_EXT}" && ln -fs "${PREFIX}/lib/libopenblas.${DYLIB_EXT}" "${PREFIX}/lib/libblas.${DYLIB_EXT}"
+test -f "${PREFIX}/lib/libopenblas.${DYLIB_EXT}" && ln -fs "${PREFIX}/lib/libopenblas.${DYLIB_EXT}" "${PREFIX}/lib/liblapack.${DYLIB_EXT}"
```

Comment:
Note that we create symlinks to the standard `blas` and `lapack` library names before the build begins. The compilers are smart enough to follow them to their true location and thus are linked to `libopenblas` directly.